File: src/main/kotlin/com/ll/nbe344team7/domain/chatroom/repository/ChatRoomRepository.java
package com.ll.nbe344team7.domain.chatroom.repository;

import com.ll.nbe344team7.domain.chatroom.entity.ChatRoom;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ChatRoomRepository extends JpaRepository<ChatRoom, Long> {
}

File: src/main/kotlin/com/ll/nbe344team7/domain/chatroom/repository/ChatRoomRepository.kt
package com.ll.nbe344team7.domain.chatroom.repository;

import com.ll.nbe344team7.domain.chatroom.entity.ChatRoom;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

/**
 *
 *
 * @author kjm72
 * @since 2025-03-26
 */
@Repository
interface ChatRoomRepository : JpaRepository<ChatRoom, Long> {
}
